dump.go

```diff
@@ -97,7 +97,14 @@ func (d *Dumper) Dump() error {
 
 func (d *Dumper) dumpTable(table string) error {
 	query := fmt.Sprintf("SELECT * FROM %s", d.helper.quoteKeyword(table))
-	rows, err := d.db.Query(query)
+
+	stmt, err := d.db.Prepare(query)
+	if err != nil {
+		return err
+	}
+	defer stmt.Close()
+
+	rows, err := stmt.Query()
```

Comment:
The currently exported yml exports the integer as a string type, such as a primary key ID
```yml
- id: "1"    # primary key 
  user_id: "1"  
  content: test
  created_at: 2019-03-06T15:05:46+08:00
  updated_at: 2019-03-06T15:05:46+08:00
```
To resolve this issue, Must use mysql Prepare,This will give you the correct type
```yml
- id: 1    # primary key 
  user_id: 1 
  content: test
  created_at: 2019-03-06T15:05:46+08:00
  updated_at: 2019-03-06T15:05:46+08:00
```

Reply:
I think that shouldn't happen anyway, but no problem